src/test/java/org/broadinstitute/hellbender/tools/PrintBGZFBlockInformationIntegrationTest.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+
+public class PrintBGZFBlockInformationIntegrationTest extends CommandLineProgramTest {
+
+    @Test
+    public void testNormalInput() throws IOException {
```

Comment:
You should probably add a test asserting that this behaves rationally for bad input (eg. ending on a non-empty block or by having null blocks interspersed with occupied blocks in the file)

Reply:
I'll add a test with a corrupt BGZF file.